mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/dao/LoggingGenerator.java

```diff
@@ -49,8 +46,12 @@ public void addLoggerField(TypeSpec.Builder classBuilder, ClassName className) {
     if (logsEnabled) {
       classBuilder.addField(
           FieldSpec.builder(
-                  LOGGER_CLASS_NAME, "LOG", Modifier.PRIVATE, Modifier.FINAL, Modifier.STATIC)
-              .initializer("$T.getLogger($T.class)", LOGGER_FACTORY_CLASS_NAME, className)
+                  ClassName.get(Logger.class),
+                  "LOG",
+                  Modifier.PRIVATE,
+                  Modifier.FINAL,
+                  Modifier.STATIC)
+              .initializer("$T.getLogger($T.class)", LoggerFactory.class, className)
```

Comment:
Not mandatory, but I figured we might as well revert this too, since SLF4J types will now be available at processing time.

Reply:
I'm unsure about this, we are importing a class just to feed it to javapoet code generator (as opposed to using it to actually log messages). The previous version seemed a bit more accurate to me.

Reply:
Either way works for me, I will revert this part.